WORK-IN-PROGRESS

```diff
@@ -474,11 +476,75 @@ Modified: 2017-08-18
     I will not provide any user support for MinGW msys/1.0 in future.
     
     MinGW msys/2.0 might be supported as "experimental" in Exiv2 v0.26.2
-     
+    
+4.9 Building and linking XMPsdk with libexiv2
+
+    Exiv2 v0.26 (and many earlier editions) contain Adobe XMPsdk code.
+    The precise version of this code is unknown.
+    The code is stored in the xmpsdk/ directory which contains src/ and include/ directories.
+    The xmpsdk has it's own Makefile, CMakeLists.txt and msvc/xmpsdk/project files.
+    
+    Exiv2 v0.26 builds a static library libxmpsdk.a which is linked into libexiv2.a
+    
+    None of this is the change in Exiv2 v0.26.1
+    
+    Exiv2 v0.26.1 will support a new option to download and build "pure Adobe" XMPsdk.
```

Comment:
Yes.  That's possible.  Adobe XMPsdk is supported on Cygwin (although I'm confident of fixing that).  The user who requested this needs Adobe XMPsdk and msvc, so I'll move in that direction.  CMake/--adobe and CMake/Cygwin are not essential to v0.26.1.  I've also realised that there isn't enough time to get v0.26.1 to GM by the end of September.  However, we'll have v0.26.1 RC1 and request feedback.